extensions/projection/README.md

```diff
@@ -52,20 +58,41 @@ If the data does not have a CRS, such as in the case of non-rectified imagery wi
-coordinates based on the CRS defined in the `proj:epsg`, `proj:projjson` or `proj:wkt2` fields.  First two numbers are coordinates 
-of the lower left corner, followed by coordinates of upper right corner, , e.g., \[west, south, east, north], 
+**proj:bbox** - Bounding box of the assets represented by this item in the asset data CRS. Specified as 4 or 6
+coordinates based on the CRS defined in the `proj:epsg`, `proj:projjson` or `proj:wkt2` fields.  First two numbers are coordinates
+of the lower left corner, followed by coordinates of upper right corner, , e.g., \[west, south, east, north],
 \[xmin, ymin, xmax, ymax], \[left, down, right, up], or \[west, south, lowest, east, north, highest]. The length of the array must be 2*n where n is the number of dimensions. The array contains all axes of the southwesterly most extent followed by all axes of the northeasterly most extent specified in Longitude/Latitude or Longitude/Latitude/Elevation based on [WGS 84](http://www.opengis.net/def/crs/OGC/1.3/CRS84). When using 3D geometries, the elevation of the southwesterly most extent is the minimum elevation in meters and the elevation of the northeasterly most extent is the maximum in meters.
 
-**proj:centroid** - Coordinates representing the centroid of the item in the asset data CRS.  Coordinates are 
+**proj:centroid** - Coordinates representing the centroid of the item in the asset data CRS.  Coordinates are
 defined in latitude and longitude, even if the data coordinate system does not use lat/long.
 
+**proj:shape** - An array of integers that represents the number of pixels in the most common pixel grid used by the item's assets.
+The number of pixels should be specified in Y, X order. If the shape is defined in an item's properties it is used as
+the default shape for all assets that don't have an overriding shape.
+
+**proj: transform** - Linear mapping from pixel coordinate space (Pixel, Line) to projection coordinate space (Xp, Yp). It is a `3x3` matrix stored as a flat array of 9 elements in row major order. Since the last row is always `0,0,1` it can be omitted, in which case only 6 elements are recorded. This mapping can be obtained from GDAL([`GetGeoTransform`](https://gdal.org/api/gdaldataset_cpp.html#_CPPv4N11GDALDataset15GetGeoTransformEPd)) or the Rasterio ([`Transform`](https://rasterio.readthedocs.io/en/stable/api/rasterio.io.html#rasterio.io.BufferedDatasetWriter.transform)).
+
+``` bash
+  [Xp]   [a0, a1, a2]   [Pixel]
+  [Yp] = [a3, a4, a5] * [Line ]
+  [1 ]   [0 ,  0,  1]   [1    ]
+```
+
+If the transform is defined in an item's properties it is used as the default transform for all assets that don't have an overriding transform.
+
+### Item [`Asset Object`](../../item-spec/item-spec.md#asset-object) fields
+
+| Field Name | Type     | Description                                  |
+| ---------- | -------- | -------------------------------------------- |
+| proj:shape    | \[number] | See item description.  |
```

Comment:
Above it's said that the shape consists of integers, so I'd propose to use \[integer].